scripts/entrypoint.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+## Inspired by https://github.com/kartoza/docker-geoserver
+set -e
+
+## install GeoServer extensions before starting the tomcat
+echo "Starting installation of extensions"
+/scripts/install-extensions.sh
```

Comment:
I would not install extensions at run time. For several reasons:
- does not work for community plugins
- it would not work in an environment without internet access
- extensions are downloaded every time the container is started. That's a lot of downloads
- should be a temporary issue with sourceforge geoserver will start without the necessary plugins and not function properly (even fail to start?)

I think GeoServer war bundling should just not happen here but instead should be handled separately and documented in the README. Users still have several options available to customize the application like bind mount a locally available war or override the command / enrtypoint to actually download extensions at startup if they want to or actually change the dockerfile and [ADD](https://docs.docker.com/engine/reference/builder/#add) or [COPY](https://docs.docker.com/engine/reference/builder/#copy) their war file directly into the docker image

Reply:
Yes. I agree that downloading and installing extensions at run time should not be the default, but i'd really like to keep this as a configurable feature (disabled by default) for quick setups.
 
I generally agree (regarding war bundling) that it should also be possible to build offline by mounting all required resources (like customized war files).

I'll work on this sooner or later and add some changes.

Reply:
I added a flag to control whether extensions should be downloaded at runtime or not. default value is false:  a4e3a83

Next i'll add a possibility to install extensions from a mounted directory.

Reply:
Maybe you also like to have a look at 4261b1b which introduces the possibility to add custom jars/libs for now.